### ISSUE TITLE
Consistent with the EU Model Accessibility Statement, use findings to generate "overall accessibility" for the Executive Summary

### ISSUE THREAD

**Issue by mitchellevan** [w3c/wcag-em-report-tool#346]
In [5. Report Findings](), instructions for the Executive Summary field suggest that users "describe the overall accessibility of the website". EU regulations provide a way for such a description to be generated automatically.

The [2018 Model Accessibility Statement](https://eur-lex.europa.eu/eli/dec_impl/2018/1523/oj) requires public sector bodies in the EU to use the following boilerplate in public-facing accessibility statements.

> (a) [This] [These] [website(s)] [mobile application(s)] [is] [are] fully compliant with [xxx].
> (b) [This] [These] [website(s)] [mobile application(s)] [is] [are] partially compliant with [xxx], due to [the non-compliance(s)] [and/or] [the exemptions] listed below.
> (c) [This] [These] [website(s)] [mobile application(s)] [is] [are] not compliant with [xxx]. The [non-compliance(s)] [and/or] [the exemptions] are listed below.

Normative footnotes specify how to choose the correct summary.

> Select (a) only if all requirements of the standard or technical specification are fully met without exceptions.
> Select (b) if most requirements of the standard or technical specification are met, but with some exceptions.
> Select (c) if most requirements of the standard or technical specification are not met.

The EU Model Accessibility Statement lists "a link to an evaluation report" as "optional content" which may be added...

> in particular if the compliance status of the website or mobile application is indicated as being ‘(a) fully compliant’

No specific language is required in the evaluation report. However, it would be strange if the Executive Summary of an EU evaluation report gave a different overall rating compared with the Accessibility Statement that linked to it.

Therefore, it would be useful to **generate a corresponding "fully compliant" / "partially compliant" / "not compliant" statement based on whether "most requirements of the standard... are met".**

Note: based on past experience with WCAG reports for many different products, condition "c" will be rare ("most requirements of the standard... are not met"). Even for products with lots of accessibility problems, it's hard to fail 50% or more of the WCAG criteria, because many criteria are not applicable to any given product.

**Comment by nitedog** [comment]
This could be done along with updating the [Report Template](https://www.w3.org/WAI/test-evaluate/report-template/), planned for 2021. cc @shawna-slh
